RadarSDK/Radar.m

```diff
@@ -53,6 +54,8 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
         [RadarSettings updateSessionId];
     }
 
+    [[RadarReplayBuffer sharedInstance] loadReplaysFromPersistentStore];
```

Comment:
don't have a strong opinion, but maybe we should load replays into memory after the call to `/config` is made and tracking options are updated

Reply:
Yea -- perhaps we gate this on `replays: all` after the config.

Reply:
I like that